proto/message.py

```diff
@@ -602,6 +602,35 @@ def __setattr__(self, key, value):
         if pb_value is not None:
             self._pb.MergeFrom(self._meta.pb(**{key: pb_value}))
 
+    def copy_from(self, other):
+        """Equivalent for protobuf.Message.CopyFrom
+
+        Args:
+            other: (Union[dict, ~.Message):
+                A dictionary or message to reinitialize the values for this message.
+        """
+        if isinstance(other, type(self)):
+            # Just want the underlying proto.
+            other = Message.pb(other)
```

Comment:
This is equivalent to doing `other = other._pb` right?

Reply:
Yes, it is equivalent. The code (and documentation) tries to use the accessor instead of touching the attribute directly.

Reply:
👍🏻